src/pages/Payment/PaymentSummary.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react';
+import { AiFillQuestionCircle } from 'react-icons/ai';
+import { priceToString } from 'utils/utilFunc';
+import './PaymentSummary.scss';
+
+const PaymentSummary = () => {
+  const [isHovered, setIsHovered] = useState(false);
+
+  const openTooltip = () => {
+    setIsHovered(true);
+  };
+
+  const closeTooltip = () => {
+    setIsHovered(false);
+  };
```

Comment:
이거 이런식으로 하셔도 될 것 같습니다. 
```suggestion
  const openTooltip = () => setIsHovered(true);

  const closeTooltip = () => setIsHovered(false);
```